src/redux/initialState.js

```diff
@@ -1225,6 +1225,31 @@ const initialState = {
         'https://images.pexels.com/photos/3757055/pexels-photo-3757055.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=650&w=940',
     },
   ],
+
+  tabs: [
```

Comment:
taby to raczej nie w reduxie, to nie jest coś co by przychodziło np. z API. Taby są generowane przez komponent rodzica, w tym przypadku komponent z produktem.